tests/test_builders/test_build_html_tocdepth.py

```diff
@@ -52,9 +54,15 @@ def test_tocdepth(app, cached_etree_parse, fname, path, check, be_found):
     check_xpath(cached_etree_parse(app.outdir / fname), fname, path, check, be_found)
 
 
+def _intradocument(nodes: Sequence[Element]) -> None:
+    for node in nodes:
```

Comment:
I don't remember now whether the nodes being checked must exist or not, but maybe add some `assert nodes` before the for loop.

Reply:
There are other checks by default that the `nodes` are a list, and that it contains at least one item.  Even so that `assert` could make sense.. thinking about it.

Reply:
(yep, added - seems to make sense for the function to be somewhat-consistent even when out-of-context)